Module/STIG/Functions.Checklist.ps1

```diff
@@ -352,14 +373,14 @@ function Get-SettingsFromMof
 
     $mofContent = Get-MofContent -ReferenceConfiguration $referenceConfiguration
 
-    return $mofContent.Where( {$PSItem.ResourceID -match $id} )
+    return $mofContent.Where( { $PSItem.ResourceID -match $id } )
```

Comment:
Remove the whitespace here.  There should only be whitespace around the operator.